cartographer_ros/cartographer_ros/node_options.cc

```diff
@@ -34,6 +36,10 @@ NodeOptions CreateNodeOptions(
       lua_parameter_dictionary->GetDouble("submap_publish_period_sec");
   options.pose_publish_period_sec =
       lua_parameter_dictionary->GetDouble("pose_publish_period_sec");
+  options.trajectory_publish_period_sec =
+      lua_parameter_dictionary->HasKey("trajectory_publish_period_sec")
```

Comment:
Consistency: Remove the default. We should rather fix all configuration to contain this explicitly.